### DISCUSSION TITLE
Problem with _get_package_constants() when running python model inference

### DISCUSSION THREAD

**Discussion by AhmedBourouis** [deepmodeling/deepmd-kit#1558] [Q&A]
i have a problem with _get_package_constants() when running python model inference to get the predicted energy, force and virial of the system
![image](https://user-images.githubusercontent.com/25748425/157645242-62de8b3d-b83d-45d4-8746-3ed97e6d9867.png)

and indeed when i checked config.items() that are read from config_file i got this: 
![image](https://user-images.githubusercontent.com/25748425/157645654-1d12d3af-7940-47bc-a37a-47dab89c396f.png)

No 'CONFIG' section! 
does anyone encountered this or have an explanation ?

**Comment by njzjz** [comment]
I've answered you in https://github.com/deepmodeling/deepmd-kit/issues/1559#issuecomment-1064463337.
